examples/20_basics/example_tabular_classification.py

```diff
@@ -40,10 +42,10 @@
 api = TabularClassificationTask(
     # To maintain logs of the run, you can uncomment the
     # Following lines
-    # temporary_directory='./tmp/autoPyTorch_example_tmp_01',
-    # output_directory='./tmp/autoPyTorch_example_out_01',
-    # delete_tmp_folder_after_terminate=False,
-    # delete_output_folder_after_terminate=False,
+    temporary_directory='./tmp/autoPyTorch_example_tmp_01',
+    output_directory='./tmp/autoPyTorch_example_out_01',
+    delete_tmp_folder_after_terminate=False,
+    delete_output_folder_after_terminate=False,
```

Comment:
If uncomment was on purpose l'd suggest we remove the lines above.

Reply:
no, it was an artefact of debugging, I have fixed it now. Thanks